### DISCUSSION TITLE
Is the ESP Web Tools 'connect' button missing from the Getting Started / Step 2: Install firmware page

### DISCUSSION THREAD

**Discussion by dmb83** [klaasnicolaas/home-assistant-glow#406] [Q&A] [answered]
Edit: tried chromium rather than chrome, and the button appeared. Sorry for clogging up the discussion with a stupid question. Looking forward to getting the meter reader working. Thanks for all your work.
____________________________

I'm new to esphome, and have only setup a BT proxy so far. I'm trying to setup a glow meter reader, but I cannot see the blue 'connect' button on the install page. I've tried several chrome based browsers. I can see the device and successfully flash firmware using ESP Web Tools on other sites. Just can't see the connect button on this one. Is it just me?

Thanks!

![espglow](https://github.com/klaasnicolaas/home-assistant-glow/assets/91012551/9367752d-4acc-4d04-ada8-8e04886b47c7)

**Comment by klaasnicolaas** [comment, marked as answer]
If your browser is not supported, you will see the following message:
`Provisioning via USB only works in browsers which support Web Serial, such as Chrome or Edge`
